### PR TITLE
Allow multiple files to be updated at once

### PR DESCRIPTION
**Pull request by Jonathanm10**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -198,12 +198,10 @@ Actions can be used for pre or post release parts.
...
-* `update-version-class`: Update the version constant in a class file.
-    * Option `class`: path to class to be updated, or fully qualified class name of the class containing the version constant
-    * Option `pattern`: optional, use to specify the string replacement pattern in your
-      version class. %version% will be replaced by the current / next version strings.
-      For example you could use `const VERSION = '%version%';`. If you do not specify
-      this option, every occurrence of the version string in the file will be replaced.
+* `files-update`: Update the version in one or multiple files. For each file to update, please provide an array with 
+    * Option `file`: path to the file to update
+    * Option `pattern`: optional, use to specify the string replacement pattern in your file. For example: 
+    `const VERSION = '%version%';`
```
Comment (jeanmonod) [inline README.md]: @Jonathanm10 This is not completely clear what the user have to configure. Maybe we should add an example into https://github.com/liip/RMT#configuration-examples WDYT?

File: src/Liip/RMT/Action/FilesUpdateAction.php [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+use Liip\RMT\Context;
+use Liip\RMT\Exception;
+use ReflectionClass;
+
+class FilesUpdateAction extends BaseAction
+{
+    public function __construct($options)
+    {
+        parent::__construct($options);
+    }
```
Comment (jeanmonod) [inline src/Liip/RMT/Action/FilesUpdateAction.php]: No need that empty constructor

**Review by jeanmonod — CHANGES_REQUESTED** [review]
@Jonathanm10 Just two comments, then for me you can merge and release a new version with that feature. Good job 👍

File: src/Liip/RMT/Action/UpdateVersionClassAction.php [hunk 3]
```diff
@@ -29,6 +29,7 @@
  * }
  *
  * @author David Buchmann <mail@davidbu.ch>
+ * @deprecated Please use UpdateVersionFilesAction instead
```
Comment (seyfahni) [inline src/Liip/RMT/Action/UpdateVersionClassAction.php]: Shouldn't it be instead:
```suggestion
 * @deprecated Please use FilesUpdateAction instead
```

Comment (jeanmonod) [reply src/Liip/RMT/Action/UpdateVersionClassAction.php]: Good catch @seyfahni !

**Pull request merged**